runelite-client/src/main/java/net/runelite/client/plugins/raids/bombs/BombOverlay.java

```diff
@@ -0,0 +1,189 @@
+/*
+ * Copyright (c) 2018, PallasDieKatze (Pallas Cat)
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *
+ * 1. Redistributions of source code must retain the above copyright notice, this
+ *    list of conditions and the following disclaimer.
+ * 2. Redistributions in binary form must reproduce the above copyright notice,
+ *    this list of conditions and the following disclaimer in the documentation
+ *    and/or other materials provided with the distribution.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
+ * ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+ * WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE FOR
+ * ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+ * (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+ * LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND
+ * ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+ * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+package net.runelite.client.plugins.raids.bombs;
+
+import lombok.extern.slf4j.Slf4j;
+import net.runelite.api.Client;
+import net.runelite.api.Perspective;
+import net.runelite.api.Player;
+import net.runelite.api.Point;
+import net.runelite.api.coords.LocalPoint;
+import net.runelite.api.coords.WorldPoint;
+import net.runelite.client.plugins.raids.RaidsConfig;
+import net.runelite.client.plugins.raids.RaidsPlugin;
+import net.runelite.client.ui.overlay.*;
+import javax.inject.Inject;
+import java.awt.*;
+import java.text.DecimalFormat;
+import java.text.NumberFormat;
+import java.time.Instant;
+import java.util.Iterator;
+import java.util.Locale;
+import java.util.Map;
+
+@Slf4j
+public class BombOverlay extends Overlay
+{
+
+	public enum Normal_Color
+	{
+		SAFE("#00cc00"), //safe
+		CAUTION("#ffff00"), //1 tile in range (minor damage)
+		WARNING("#ff9933"), //2 tiles in range (moderate damage)
+		DANGER("#ff6600"), //3 tiles in range/adjacent to bomg (major damage)
+		LETHAL("#cc0000"); //On the bomb, using it as a makeshift space launch vehicle. (massive damage)
+
+		private String hex;
```

Comment:
Just store this as Color. Or maybe just makes these colors all final/static instead of an enum.